ui/apps/platform/src/utils/searchUtils.ts

```diff
@@ -116,6 +116,25 @@ export function convertSortToRestFormat(graphqlSort: GraphQLSortOption[]): Parti
     };
 }
 
+/**
+ * Function to convert the legacy SearchEntry array format to the
+ * SearchFilter format.
+ */
+export function searchOptionsToSearchFilter(searchOptions: GlobalSearchOption[]): SearchFilter {
```

Comment:
This logic seems like it must exist elsewhere, but I could not find it via search in VSCode.

If there is an existing function that converts `SearchEntry[]` to `SearchFilter` I can delete this and reference existing code.